maintenance_page/manifests/maintenance/deployment_maintenance.yml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1
```

Comment:
this file not required, it's created by the script.

Reply:
corrected. removed.